src/seth/libexec/seth/seth

```diff
@@ -59,6 +59,9 @@ N,nonce=number       nonce value to use for the transaction
 V,value=number       amount (in wei) of ETH to send
 S,password=file      password for non-interactive signing
 
+  Options for seth-block(1):
```

Comment:
Why these `(1)` at the end? Looks like manpage indexes but this not being a manpage, I'm confused.

Reply:
I dunno really, I was just following the pattern for the other commands